From f545f2ec1ffcaee32d7daa3f2210bf487da4a1d5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 12 Apr 2023 11:39:33 +0000
Subject: [PATCH 19/23] [compiler-rt] [test] [profile] Mark the without-libc
 test as unsupported on mingw

Mingw toolchains always end up referencing the malloc symbol due to
the CRT startup files.
---
 compiler-rt/test/profile/instrprof-without-libc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/compiler-rt/test/profile/instrprof-without-libc.c b/compiler-rt/test/profile/instrprof-without-libc.c
index b7f7a1e9c688..bd7b925ab9cb 100644
--- a/compiler-rt/test/profile/instrprof-without-libc.c
+++ b/compiler-rt/test/profile/instrprof-without-libc.c
@@ -9,6 +9,9 @@
 // an MSVC environment, so we can't make this test portable.
 // UNSUPPORTED: target={{.*msvc.*}}
 
+// The mingw CRT init files do reference malloc etc, so this test fails.
+// UNSUPPORTED: target={{.*windows-gnu.*}}
+
 #include <stdint.h>
 #include <stdlib.h>
 
-- 
2.34.1

